### ISSUE TITLE
Unable to Record

### ISSUE THREAD

**Issue by nowjon** [webrecorder/webrecorder#792]
Hi everyone, 

I just installed the docker containers with the Docker Compose and it's all running but I'm unable to capture any websites. 

It kind of just sits there, 
![image](https://user-images.githubusercontent.com/40905072/79704528-8c03cd80-8277-11ea-817f-6ab13997004e.png)

I see errors in the browser DevTools, 
![image](https://user-images.githubusercontent.com/40905072/79704581-b9507b80-8277-11ea-83b0-6779740e5b60.png)

Here's the output of the "sudo docker-compose logs app" command: 
`Attaching to app
app           | [uWSGI] getting INI configuration from /code/apps/apiapp.ini
app           | [uwsgi-static] added mapping for http://webrecorder.proxy/static => /code/webrecorder/static
app           | [uwsgi-static] added mapping for /static => /code/webrecorder/static
app           | *** Starting uWSGI 2.0.18 (64bit) on [Mon Apr 20 00:37:53 2020] ***
app           | compiled with version: 6.3.0 20170516 on 21 February 2020 06:00:24
app           | os: Linux-5.3.0-7642-generic #34~1584408018~19.10~21df4b1-Ubuntu SMP Tue Mar 17 13:38:51 UTC
app           | nodename: d3cddf1e8e2f
app           | machine: x86_64
app           | clock source: unix
app           | pcre jit disabled
app           | detected number of CPU cores: 8
app           | current working directory: /code
app           | detected binary path: /usr/local/bin/uwsgi
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | your memory page size is 4096 bytes
app           | detected max file descriptor number: 65536
app           | building mime-types dictionary from file /etc/mime.types...554 entry found
app           | - async cores set to 400 - fd table size: 65536
app           | lock engine: pthread robust mutexes
app           | thunder lock: disabled (you can enable it with --thunder-lock)
app           | uwsgi socket 0 bound to TCP address :8081 fd 3
app           | uwsgi socket 1 bound to TCP address :8088 fd 4
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | Python version: 3.7.2 (default, Mar 27 2019, 08:41:08)  [GCC 6.3.0 20170516]
app           | Python main interpreter initialized at 0x5555b324fca0
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | python threads support enabled
app           | your server socket listen backlog is limited to 100 connections
app           | your mercy for graceful operations on workers is 60 seconds
app           | mapped 317025104 bytes (309594 KB) for 4000 cores
app           | *** Operational MODE: preforking+async ***
app           | /usr/local/lib/python3.7/site-packages/webassets/loaders.py:201: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
app           |   obj = self.yaml.load(f) or {}
app           | No Archives Loaded
app           | skip {'name': 'Admin', 'description': 'Admin API'}
app           | skip {'name': 'Stats', 'description': 'Stats API'}
app           | skip {'name': 'Automation', 'description': 'Automation API'}
app           | WSGI app 0 (mountpoint='') ready in 10 seconds on interpreter 0x5555b324fca0 pid: 1 (default app)
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | *** uWSGI is running in multiple interpreter mode ***
app           | spawned uWSGI master process (pid: 1)
app           | spawned uWSGI worker 1 (pid: 14, cores: 400)
app           | spawned uWSGI worker 2 (pid: 15, cores: 400)
app           | spawned uWSGI worker 3 (pid: 16, cores: 400)
app           | *** running gevent loop engine [addr:0x5555b1a37fd0] ***
app           | spawned uWSGI worker 4 (pid: 17, cores: 400)
app           | spawned uWSGI worker 5 (pid: 18, cores: 400)
app           | spawned uWSGI worker 6 (pid: 20, cores: 400)
app           | /usr/local/lib/python3.7/site-packages/webassets/loaders.py:201: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
app           |   obj = self.yaml.load(f) or {}
app           | spawned uWSGI worker 7 (pid: 22, cores: 400)
app           | spawned uWSGI worker 8 (pid: 23, cores: 400)
app           | spawned uWSGI worker 9 (pid: 24, cores: 400)
app           | spawned uWSGI worker 10 (pid: 27, cores: 400)
app           | Updating bundle/base.js
app           | Updating bundle/base.css
app           | Updating bundle/fonts.css
app           | Updating bundle/main.js
app           | Updating bundle/main.css
app           | Updating bundle/index.js
app           | Updating bundle/index.css
app           | Updating bundle/wb_frame.js
app           | /usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py:1145: DeprecationWarning: Use of .. or absolute path in a resource path is not allowed and will raise exceptions in a future release.
app           |   self, resource_name
app           | Updating bundle/frame.js
app           | Updating bundle/frame.css
app           | Updating bundle/coll.js
app           | Updating bundle/coll.css
app           | Updating bundle/pywb.js
app           | Updating bundle/wombatWorkers.js
app           | Updating bundle/autoFetchWorker.js
app           | Updating bundle/proxy.js
app           | Updating bundle/browser.js
app           | Updating bundle/browser.css
app           | Updating bundle/paging.js
app           | Mon Apr 20 00:40:33 2020 - uWSGI worker 8 screams: UAAAAAAH my master disconnected: i will kill myself !!!
app           | Mon Apr 20 00:40:33 2020 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
app           | Mon Apr 20 00:40:33 2020 - uWSGI worker 6 screams: UAAAAAAH my master disconnected: i will kill myself !!!
app           | [uWSGI] getting INI configuration from /code/apps/apiapp.ini
app           | [uwsgi-static] added mapping for http://webrecorder.proxy/static => /code/webrecorder/static
app           | [uwsgi-static] added mapping for /static => /code/webrecorder/static
app           | *** Starting uWSGI 2.0.18 (64bit) on [Mon Apr 20 00:40:45 2020] ***
app           | compiled with version: 6.3.0 20170516 on 21 February 2020 06:00:24
app           | os: Linux-5.3.0-7642-generic #34~1584408018~19.10~21df4b1-Ubuntu SMP Tue Mar 17 13:38:51 UTC
app           | nodename: d3cddf1e8e2f
app           | machine: x86_64
app           | clock source: unix
app           | pcre jit disabled
app           | detected number of CPU cores: 8
app           | current working directory: /code
app           | detected binary path: /usr/local/bin/uwsgi
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | your memory page size is 4096 bytes
app           | detected max file descriptor number: 65536
app           | building mime-types dictionary from file /etc/mime.types...554 entry found
app           | - async cores set to 400 - fd table size: 65536
app           | lock engine: pthread robust mutexes
app           | thunder lock: disabled (you can enable it with --thunder-lock)
app           | uwsgi socket 0 bound to TCP address :8081 fd 3
app           | uwsgi socket 1 bound to TCP address :8088 fd 4
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | Python version: 3.7.2 (default, Mar 27 2019, 08:41:08)  [GCC 6.3.0 20170516]
app           | Python main interpreter initialized at 0x5628e81c7ca0
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | python threads support enabled
app           | your server socket listen backlog is limited to 100 connections
app           | your mercy for graceful operations on workers is 60 seconds
app           | mapped 317025104 bytes (309594 KB) for 4000 cores
app           | *** Operational MODE: preforking+async ***
app           | /usr/local/lib/python3.7/site-packages/webassets/loaders.py:201: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
app           |   obj = self.yaml.load(f) or {}
app           | No Archives Loaded
app           | skip {'name': 'Admin', 'description': 'Admin API'}
app           | skip {'name': 'Stats', 'description': 'Stats API'}
app           | skip {'name': 'Automation', 'description': 'Automation API'}
app           | WSGI app 0 (mountpoint='') ready in 6 seconds on interpreter 0x5628e81c7ca0 pid: 1 (default app)
app           | uWSGI running as root, you can use --uid/--gid/--chroot options
app           | *** WARNING: you are running uWSGI as root !!! (use the --uid flag) ***
app           | *** uWSGI is running in multiple interpreter mode ***
app           | spawned uWSGI master process (pid: 1)
app           | spawned uWSGI worker 1 (pid: 14, cores: 400)
app           | spawned uWSGI worker 2 (pid: 15, cores: 400)
app           | spawned uWSGI worker 3 (pid: 16, cores: 400)
app           | spawned uWSGI worker 4 (pid: 17, cores: 400)
app           | *** running gevent loop engine [addr:0x5628e6a4cfd0] ***
app           | spawned uWSGI worker 5 (pid: 18, cores: 400)
app           | spawned uWSGI worker 6 (pid: 19, cores: 400)
app           | /usr/local/lib/python3.7/site-packages/webassets/loaders.py:201: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
app           |   obj = self.yaml.load(f) or {}
app           | spawned uWSGI worker 7 (pid: 22, cores: 400)
app           | spawned uWSGI worker 8 (pid: 24, cores: 400)
app           | spawned uWSGI worker 9 (pid: 27, cores: 400)
app           | spawned uWSGI worker 10 (pid: 28, cores: 400)
app           | /usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py:1145: DeprecationWarning: Use of .. or absolute path in a resource path is not allowed and will raise exceptions in a future release.
app           |   self, resource_name
app           | [pid: 14|app: 0|req: 1/1] 172.18.0.9 () {40 vars in 1176 bytes} [Mon Apr 20 00:41:40 2020] GET /api/v1/auth/curr_user?include_colls=true => generated 306 bytes in 6 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 28|app: 0|req: 1/2] 10.10.10.122 () {52 vars in 1481 bytes} [Mon Apr 20 00:42:31 2020] POST /api/v1/auth/login?include_colls=true => generated 926 bytes in 18 msecs (HTTP/1.1 200) 9 headers in 778 bytes (3 switches on core 399)
app           | [pid: 28|app: 0|req: 2/3] 10.10.10.122 () {46 vars in 949 bytes} [Mon Apr 20 00:42:31 2020] GET /api/v1/collections?user=nowjon&include_pages=false&include_recordings=false&include_lists=false => generated 546 bytes in 172 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 24|app: 0|req: 1/4] 10.10.10.122 () {46 vars in 835 bytes} [Mon Apr 20 00:42:31 2020] GET /api/v1/user/nowjon?include_colls=false => generated 397 bytes in 173 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 28|app: 0|req: 3/5] 10.10.10.122 () {46 vars in 804 bytes} [Mon Apr 20 00:42:31 2020] GET /api/v1/client_archives => generated 2 bytes in 4 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 2/6] 10.10.10.122 () {46 vars in 874 bytes} [Mon Apr 20 00:42:35 2020] GET /api/v1/collection/default-collection?user=nowjon => generated 672 bytes in 12 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 3/7] 10.10.10.122 () {46 vars in 918 bytes} [Mon Apr 20 00:42:35 2020] GET /api/v1/lists?user=nowjon&coll=default-collection&include_bookmarks=all => generated 13 bytes in 6 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 4/8] 10.10.10.122 () {46 vars in 881 bytes} [Mon Apr 20 00:42:37 2020] GET /api/v1/collection/default-collection?user=nowjon => generated 672 bytes in 8 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 5/9] 10.10.10.122 () {46 vars in 828 bytes} [Mon Apr 20 00:42:38 2020] GET /api/v1/client_archives => generated 2 bytes in 4 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 399)
app           | [pid: 19|app: 0|req: 1/11] 172.18.0.9 () {40 vars in 1641 bytes} [Mon Apr 20 00:48:12 2020] GET /api/v1/auth/curr_user?include_colls=true => generated 943 bytes in 227 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 6/11] 172.18.0.9 () {40 vars in 1657 bytes} [Mon Apr 20 00:48:12 2020] GET /api/v1/collection/default-collection?user=nowjon => generated 672 bytes in 8 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 7/12] 172.18.0.9 () {40 vars in 1641 bytes} [Mon Apr 20 00:48:13 2020] GET /api/v1/auth/curr_user?include_colls=true => generated 943 bytes in 8 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 8/13] 172.18.0.9 () {40 vars in 1657 bytes} [Mon Apr 20 00:48:13 2020] GET /api/v1/collection/default-collection?user=nowjon => generated 672 bytes in 6 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 399)
app           | [pid: 14|app: 0|req: 9/14] 10.10.10.122 () {46 vars in 1837 bytes} [Mon Apr 20 00:48:14 2020] GET /api/v1/client_archives => generated 2 bytes in 4 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 399)`

I have not really changed anything from the default. 

Anyone have any ideas as to why I don't get a browser?

**Comment by m4rk3r** [comment]
Hi @nowjon do you have any browser extensions that might be blocking traffic, like privacy badger?

**Comment by nowjon** [comment]
I found the issue out, sorry for not updating. 

I was attempting from another machine on the network instead of localhost. I can access it from my workstation now but I had to change the two env variables


PUBLIC_IP | 127.0.0.1
APP_HOST | 10.10.10.240:8089
CONTENT_HOST | 10.10.10.240:8092

(i.e. edit the APP_HOST & CONTENT_HOST to the server IP and their ports, the above are the default ports. 

Hope this helps out anyone else
